odd-platform-specification/components.yaml

```diff
@@ -664,6 +664,8 @@ components:
       properties:
         name:
           type: string
+        namespace_name:
```

Comment:
why is it the name, not the id, btw?

Reply:
We want to create the namespace if it's missing and do that in one request. I understand it's arguable and we had our hiccups here during the discussion, but right now it's the `by design`. Feel free to raise this question internally